examples/beam/src/main/java/org/apache/nemo/examples/beam/AlternatingLeastSquare.java

```diff
@@ -268,8 +268,8 @@ public void processElement(final ProcessContext c) throws Exception {
     extends LoopCompositeTransform<PCollection<KV<Integer, float[]>>, PCollection<KV<Integer, float[]>>> {
     private final Integer numFeatures;
     private final double lambda;
-    private final PCollection<KV<Integer, KV<int[], float[]>>> parsedUserData;
-    private final PCollection<KV<Integer, KV<int[], float[]>>> parsedItemData;
+    private final PCollection<KV<Integer, KV<int[], float[]>>> parsedUserData; //NOSONAR
+    private final PCollection<KV<Integer, KV<int[], float[]>>> parsedItemData; //NOSONAR
```

Comment:
For these lines, let's declare the variables `transient`, instead of putting `//nosonar` here. In that case, you would have to declare `private final transient PCollection...` for both lines. Thanks!